graph/patterns/dictionary.md

```diff
@@ -41,15 +41,23 @@ For more information, see the [OData reference](https://github.com/oasis-tcs/oda
 The following example demonstrates defining a dictionary that can contain string values.
 
 ```xml
-<ComplexType Name="stringDictionary" BaseType="graph.Dictionary">
-  <Annotation Term="Org.OData.Validation.V1.OpenPropertyTypeConstraint">
-    <Collection>
-      <String>Edm.String</String>
-    </Collection>
-  </Annotation>
-</ComplexType>
+<Schema Namespace="WorkloadNamespace">
```

Comment:
I'm good with the change, but kind of feel like it needs a minor restructure.
Under Examples, we should have 2 sub-sections - String dictionary and Complex type dictionary and then put the current relevant sections under those two.

Reply:
I created a subsection for string vs complex type dictionary